### ISSUE TITLE
QuestGUI : choose is not a dcommand

### ISSUE THREAD

**Issue by Oruss7** [AnthonyAMC/Public-Denizen-Scripts#62]
Got errors when openning Gui (with quest book).

is installed:  
- message constructor library
- quest GUI
- example quest for quest GUI

Have this error on my console: 
![log](https://cloud.githubusercontent.com/assets/4788317/14760786/03cf37c6-094d-11e6-9503-84e9efc037c2.PNG)

Using version: 
```
#   Authors: |Anthony|
#   Version: 0.1
#   dScript Version: 0.9.8-DEV_b548
```

and denizen `Denizen v0.9.6-SNAPSHOT (build 1584)`

Others info: 
- run on win7
- encoding with ANSI
- No tabulation, 4 spaces
- No problem when /denizen reload

**Comment by mcmonkey4eva** [comment]
That script requires Minecraft 1.8.8 or more recent to function.

**Comment by Oruss7** [comment]
i run spigot 1.8.8

**Comment by mcmonkey4eva** [comment]
Then I'm unsure how you're running Denizen 0.9.6, but okay.
Here, try running 0.9.7 instead: http://ci.citizensnpcs.co/job/Denizen/1610/

**Comment by Oruss7** [comment]
Thanks, problem resolved with version 0.9.7.
